crates/router/src/connector/opennode.rs

```diff
@@ -72,6 +72,10 @@ impl ConnectorCommon for Opennode {
         "opennode"
     }
 
+    fn get_currency_unit(&self) -> api::CurrencyUnit {
+        api::CurrencyUnit::Base
+    }
```

Comment:
The CurrenyUnit will be `Minor` in this case as per open node requirements

```suggestion
        api::CurrencyUnit::Minor
    }
```

Reply:
Modified!